api/capi/include/nnstreamer.h

```diff
@@ -1069,6 +1069,9 @@ int ml_tensors_data_create (const ml_tensors_info_h info, ml_tensors_data_h *dat
 
 /**
  * @brief Frees the given tensors' data handle.
+ * @details Note that this should be called before destroying the inference data by ml_tensors_data_destroy().
```

Comment:
???? Clarify This Sentence... ????

You need to call ```ml_pipeline_if_custom``` before calling ```ml_pipeline_if_custom``` ???

Reply:
Hmm? I couldn't see such sentences. Maybe misreading?

Reply:
My falut. I added the note section to both `ml_tensors_data_destroy()` and `ml_single_close()`.